### ISSUE TITLE
Doesn't work with Pinned Tab

### ISSUE THREAD

**Issue by nengqi** [brcontainer/prevent-duplicate-tabs#20]
**Describe the bug**
open a tab and pin it, then open another one with the same address, neither one of them was closed

**Comment by brcontainer** [comment]
@nengqi pinned tabs are disregarded, precisely because pinned tabs are important and will not be considered.

**Comment by K0-RR** [comment]
I also have this issue. I don't understand your explanation. Just close the not pinned one, or at least display a pop-up saying the duplicate wasn't closed.

**Comment by brcontainer** [comment]
@K0-RR Hi! Pinned tabs are not considered, so unpinned duplicate tabs can be created without triggering the search and checking for new tabs.

If you have a suggestion for improving the descriptive texts on pinned tabs in the addon, for better understanding, you can send it to me, and I will translate and add it in the supported languages. Thanks.
